src/arrayMethodJoin.js

```diff
@@ -4,8 +4,29 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let string = '';
```

Comment:
It's good to use more informative names for variables, _e.g. joinedString_

Reply:
Not fixed, useful tip, naming variables more descriptively is important